pkgs/servers/mautrix-signal/default.nix

```diff
@@ -7,22 +7,25 @@
   libsignal-ffi,
   versionCheckHook,
   # This option enables the use of an experimental pure-Go implementation of
-  # the Olm protocol instead of libolm for end-to-end encryption. Using goolm
-  # is not recommended by the mautrix developers, but they are interested in
-  # people trying it out in non-production-critical environments and reporting
-  # any issues they run into.
-  withGoolm ? false,
+  # the Olm protocol instead of libolm for end-to-end encryption.
+  # While goolm has not been audited yet and is not enabled by default upstream,
+  # libolm is deprecated by its upstream and has a known security vulnerability,
+  # thus blocking hydra builds.
+  withGoolm ? true,
```

Comment:
We should *not* be defaulting to goolm. It doesn't work. The goolm support is there as an experimental option that should not be used.

See https://github.com/mautrix/go/issues/280 for an example of a bug found when using goolm.

Reply:
I'm tagged in that issue, and for what it's worth, I wasn't able to reproduce it ever since, and have been successfully running my own instance using goolm for the past few months.

Reply:
I removed the default change for now, but would be happy to help in any way I can to get either goolm to a production ready state or integrate vodozemac